functions/__tests__/functions.test.ts

```diff
@@ -23,11 +23,9 @@ describe('extension', () => {
   // Mocking of Meilisearch package
   const mockedMeilisearch = mocked(MeiliSearch, true)
   const mockedAddDocuments = jest.fn()
-  const mockedUpdateDocuments = jest.fn()
```

Comment:
By using updateDocuments instead of addDocuments the user was not able to delete fields. 

```
{
 id: 1
 other: 2
 }
```

```
updatedDocuments[{ id: 1}]
```

Does not remove the `other` field.

It may be better explained in the body of this PR